src/codeLens/prometheusScraper.ts

```diff
@@ -87,23 +97,43 @@ export class PrometheusCodeLensProvider implements vscode.CodeLensProvider {
         // Action lens
         this.codeLenses.push(
           new vscode.CodeLens(range, {
-            title: "Scrape data",
+            title: "Scrape data from endpoint",
             tooltip: "Connect to an exporter and scrape metrics, then use them in the Extension.",
             command: "dt-ext-copilot.codelens.scrapeMetrics",
             arguments: [],
           }),
         );
+        if (!this.fileScraped) {
+          this.codeLenses.push(
```

Comment:
Don't push another lens.
Use the existing "Scrape data" lens and fork the workflow behind it into "From endpoint" and "From file". 

What if we come up with 10 more ways of scraping data? Keep the screen simple.